apps/sms/views/shared/style/composer.css

```diff
@@ -4,14 +4,22 @@
 
   height: auto; /* overrides building blocks */
   min-height: 4rem;
-
   background-color: #fff;
 }
 
-/* Overwrite. No more fixed height */
-form[role="search"] p input,
-form[role="search"] p textarea,
-form[role="search"] button[type="submit"] + p > textarea {
+/* 
+ * https://bugzilla.mozilla.org/show_bug.cgi?id=1055357
+ * Instead of sticking to id selector I added a class to compose form,
+ * that holds styles which were related before to form[role="search"]
+ */
+
+.messages-compose {
+  z-index: 3;
+}
```

Comment:
I think this is useless as well now. I think this is useless since we switched to a "flex" layout